swagger/swagger_model.go

```diff
@@ -224,18 +224,19 @@ type PathItem struct {
 
 // Operation is https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object
 type Operation struct {
-	Tags         []string               `json:"tags,omitempty"`
-	Summary      string                 `json:"summary,omitempty"`
-	Description  string                 `json:"description,omitempty"`
-	ExternalDocs *ExternalDocumentation `json:"externalDocs,omitempty"`
-	OperationID  string                 `json:"operationId,omitempty"`
-	Consumes     []string               `json:"consumes,omitempty"`
-	Produces     []string               `json:"produces,omitempty"`
-	Parameters   []*Parameter           `json:"parameters,omitempty"`
-	Responses    Responses              `json:"responses" swagger:",req"`
-	Schemes      []string               `json:"schemes,omitempty"`
-	Deprecated   bool                   `json:"deprecated,omitempty"`
-	Security     []SecurityRequirement  `json:"security,omitempty"`
+	Tags                []string               `json:"tags,omitempty"`
+	Summary             string                 `json:"summary,omitempty"`
+	Description         string                 `json:"description,omitempty"`
+	ExternalDocs        *ExternalDocumentation `json:"externalDocs,omitempty"`
+	OperationID         string                 `json:"operationId,omitempty"`
+	Consumes            []string               `json:"consumes,omitempty"`
+	Produces            []string               `json:"produces,omitempty"`
+	IsSortingParameters bool                   `json:"isSortingParameters,omitempty"`
```

Comment:
this parameter is not exists on Swagger specs.
I think that it can be fixed behavior in this changes.

Reply:
Thanks for confirm !
I deleted IsSortingParameters and changed to always sorting.